### DISCUSSION TITLE
自定义组件的如何与jsonSchema中的value所绑定

### DISCUSSION THREAD

**Discussion by Amoslb** [alibaba/formily#2098] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/22581560/131946700-95dd2da9-ac9f-403d-a717-e007836a61b4.png)![image](https://user-images.githubusercontent.com/22581560/131964699-38244433-dfda-4e22-a403-152b797fc200.png). 用户尾数那里是自定义的一个组件。如何让自定义的这个组件里面的一个值，与schema对应

**Comment by janryWang** [comment]
任何一个自定义组件支持value/onChange属性就能从props上拿到最新的数据

**Reply by Amoslb** [reply]
现在拿到的是input里面的属性值。但是想获取，每次已经输入完毕后，获取点击确定的属性值，应该如何做。![企业微信截图_0aa969eb-ff19-41ae-bd3f-479ff0275afd](https://user-images.githubusercontent.com/22581560/131965287-689c9a30-bade-4398-8375-622b5349a108.png)。也就是输入框上面带x号的值。

**Reply by rayy-li** [reply]
触发change事件 `[$]emit('change')`

**Reply by celinezhi** [reply]
> 触发change事件 `[$]emit('change')`

求问触发change事件，哪里可以监听到？

**Reply by celinezhi** [reply]
> 任何一个自定义组件支持value/onChange属性就能从props上拿到最新的数据

不过onChange事件没有返回字段路径~这里大佬怎么处理呢？
